docker-compose-mongodb.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.5"
```

Comment:
nå har vi tre version af mongodb service deklarert. de to andre deklarasjoner er identiske men den her er annerledes. hvis vi klarer ikke å konsolidere de på en måte, så må vi minst ha dem identiske. (admin/admin)

Reply:
Ja, det kan vi få til.

I tillegg kan jeg endre docker-compose versjon til 3, som er i bruk ellers.

Og hvor fant jeg hvilke porter som var tatt av andre deler av fdk? Var det 8138 jeg skulle bruke?

Reply:
Rebaset inn din commit, så da skal det være admin/admin